samcli/commands/init/__init__.py

```diff
@@ -231,6 +231,12 @@ def wrapped(*args, **kwargs):
     """ """,
     required=False,
 )
+@click.option(
+    "--tracing/--no-tracing",
+    default=None,
+    help="Enable AWS X-Ray tracing for your lambda functions",
+    cls=ClickMutex,
```

Comment:
I don't think we need to use `ClickMutex` class, right? Please consider removing this line